### PR TITLE
[QC-527] - Store repoCleaner config in consul

### PR DESCRIPTION
**Pull request by Barthelemy**
Depends on https://github.com/alisw/alidist/pull/2967

### CODE REVIEW

File: Framework/script/RepoCleaner/repoCleaner.py [hunk 1]
```diff
@@ -128,6 +129,25 @@ def downloadConfigFromGit():
...
+    Download a config file from consul.
+    :return: the path to the config file
+    """
+
+    logging.debug("Get it from consul")
+    consul_server = consul.Consul(host=consul_url, port=consul_port)
+    index, data = consul_server.kv.get(key='folder/blabla')
+    logging.debug(f"config file from consul : \n{data['Value']}")
+    text = data["Value"].decode()
+    logging.debug(f"config file from consul : \n{text}")
```
Comment (knopers8) [inline Framework/script/RepoCleaner/repoCleaner.py:143]: Just a question: how are the supposed to be different?

File: Framework/script/RepoCleaner/repoCleaner.py [hunk 2]
```diff
@@ -128,6 +129,25 @@ def downloadConfigFromGit():
...
+    path = "/tmp/config.yaml"
```
Comment (knopers8) [inline Framework/script/RepoCleaner/repoCleaner.py:144]: Please use a more meaningful name (e.g. `/tmp/repoCleanerConfig.yaml`)

File: Framework/script/RepoCleaner/repoCleaner.py (hunk 1 above)
Comment (Barthelemy) [reply Framework/script/RepoCleaner/repoCleaner.py:143]: the data returned by Consul is encoded in base64 for some reason.

File: Framework/script/RepoCleaner/repoCleaner.py (hunk 2 above)
Comment (Barthelemy) [reply Framework/script/RepoCleaner/repoCleaner.py]: done

**Review by knopers8 — APPROVED** [review]
Thank you

**Pull request merged**